ui_widgets.py

```diff
@@ -143,6 +143,27 @@ def validateOut(self, event):
             self.focus_set()
 
 
+class EntryHelp:
+    def __init__(self, master, text):
+        self.master = master
+        self.text = text
+
```

Comment:
All help buttons except for the very first one say "To be added".
![image](https://github.com/Leading-AI-Development-Solutions/MAIA/assets/97774046/0bbe04ee-9589-47ad-9669-48f3465d9657)

Reply:
This is expected. Zach said that they didn't have to all be filled in, but also it might make more sense to fill them in during the documentation process.